gocov/test.go

```
@@ -45,3 +146,66 @@ func runTests(args []string) error {
 	}
 	return convertProfiles(coverprofile.Name())
 }
+
+func runAllTests(args []string) (err error) {
+	pkgRoot, _ := os.Getwd()
```

Please don't swallow errors.